bin/worker.js

```diff
@@ -212,10 +237,12 @@ async function main () {
 
   config.volumeCache = new VolumeCache(config);
 
-  config.gc.on('gc:container:removed', function (container) {
+  config.gc.on('gc:container:removed', async function (container) {
     container.caches.forEach(async (cacheKey) => {
       await config.volumeCache.release(cacheKey);
     });
+
+    await config.volumeCache.doPurge();
```

Comment:
Hrm, does this need to be done here?  I think that all of the purging stuff could just happen when the GC cycle starts.  the GC could call the volume manager and it can clear up whatever it needs to, including caches that should be purged.  As long as caches marked as "purge: true" don't get reused for a task, I don't see the harm in just deferring it to GC